Stacking

RenardQueenston opened the pull request:
:cl:
rscadd: Добавлены спрайты для бинтов, мазей и advanced патчей, отображающие их количество. Вернул возможность накладывать шину на голову, тело и гроин.
/:cl:
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->

Commit:
Merge pull request #5 from AshtonFox/dev

merge

Commit:
Update torch6_bridge.dmm

:cl:
maptweak:  Переместил конверт envelope из кабинета АВД в кабинет Представителя, АВД добавил его конверт, но от НТ; поменял кружку в кабинете АВД на кружку НТ и добавить кружку SCG в кабинет Представителя
/:cl:

Commit:
Merge pull request #6 from AshtonFox/dev

merge

Commit:
Правка описаний и цепи командования 2.0

Commit:
Update exploration_jobs.dm

Commit:
Update maps/torch/job/command_jobs.dm

Co-authored-by: AshtonFox <31184078+AshtonFox@users.noreply.github.com>

Commit:
Update maps/torch/job/command_jobs.dm

Co-authored-by: AshtonFox <31184078+AshtonFox@users.noreply.github.com>

Commit:
Update maps/torch/job/exploration_jobs.dm

Commit:
Merge pull request #7 from AshtonFox/dev

Правка описаний и цепи командования 2.0 (#27)

Commit:
Merge pull request #8 from AshtonFox/dev

merge

Commit:
Stacking

:cl:
rscadd: Добавлены спрайты для бинтов, мазей и advanced патчей, отображающие их количество. Вернул возможность накладывать шину на голову, тело и гроин.
:cl:

RenardQueenston commented:
Ну и соответственно впилена функция отображения этих спрайтов. Странно, что на ориг-торче этого нет... Думаю стоит им скинуть, чтобы не было конфликтов.

AshtonFox commented:
Ага, и будет как в прошлый раз. Но можешь попробовать, конечно же. Но для этого тебе нужен форк Бэя.

RenardQueenston commented:
Форк бея есть в организации, так что это не проблема. 

...наверное

Commit:
Разряженный пульсач.дми

:cl:
rscadd: Добавил спрайт разряженного пульсача. Почему-то его не было.
/:cl:

AshtonFox commented on code/game/objects/items/stacks/medical.dm:
```diff
@@ -247,6 +253,8 @@
 			use(1)
 			affecting.salve()
 			affecting.disinfect()
+			playsound(src, pick(apply_sounds), 25)
```
Приведёт к повторному проигрыванию звука после завершения операции, что нам, конечно же, не нужно _(см.  наличие того же процесса в строке 246)_
```suggestion
```

RenardQueenston replied:
Почему вообще не перенести проигрывание звука на конец операции..

RenardQueenston replied:
Я перенёс, уберите это как-нибудь.

AshtonFox commented on frontier/code/game/objects/items/stacks/medical_vr.dm:
```diff
@@ -0,0 +1,75 @@
+/obj/item/stack/medical/advanced
+	icon = 'icons/obj/medical_kits.dmi'
```
Дубликат переменной, заданной на более высоком уровне, без изменений. Зачем?
```suggestion
```

AshtonFox commented on frontier/code/game/objects/items/stacks/medical_vr.dm:
```diff
@@ -0,0 +1,75 @@
+/obj/item/stack/medical/advanced
```
```suggestion
```

Commit:
Отображение заряда для станрайфла

:cl:
rscadd: Отображение заряда станрайфла.
/:cl:

Co-Authored-By: Nachtjager <nachtjager@users.noreply.github.com>

AshtonFox commented on frontier/code/game/objects/items/stacks/medical_vr.dm:
```diff
@@ -0,0 +1,75 @@
+			icon_state = "[initial(icon_state)]_10"
+
+/obj/item/stack/medical/bruise_pack
+	icon = 'icons/obj/medical_kits.dmi'
```
То же, что и вверху.
```suggestion
```

AshtonFox commented on frontier/code/game/objects/items/stacks/medical_vr.dm:
```diff
@@ -0,0 +1,75 @@
+		else
+			icon_state = "[initial(icon_state)]_10"
+
+/obj/item/stack/medical/bruise_pack
```
```suggestion
```

AshtonFox commented on frontier/code/game/objects/items/stacks/medical_vr.dm:
```diff
@@ -0,0 +1,75 @@
+			icon_state = "[initial(icon_state)]_10"
+
+/obj/item/stack/medical/ointment
+	icon = 'icons/obj/medical_kits.dmi'
```
```suggestion
```

AshtonFox commented on frontier/code/game/objects/items/stacks/medical_vr.dm:
```diff
@@ -0,0 +1,75 @@
+		else
+			icon_state = "[initial(icon_state)]_10"
+
+/obj/item/stack/medical/ointment
```
```suggestion
```

RenardQueenston replied:
Нахуя

AshtonFox replied:
Прочти остальные, что выше, это одна и та же проблема.

RenardQueenston replied:
Пойду на локалку тестить опять

Я не верю, что кодеры Воре просто так это написали

Commit:
Update medical.dm

:cl:
tweak: Мазей и бёрн-паков теперь 10, вместо 5, а звуки проигрываются только после успешного наложения мази или бинта. Дубликаты убрал.
/:cl:

Commit:
Перенос спрайтов

:cl:
tweak: Спрайты из `medical_kits.dmi` в `medical_kits_vr.dmi`
/:cl:
